Data/models/DataController.swift

```diff
@@ -45,6 +45,13 @@ public class DataController: NSObject {
     public static var shared: DataController = DataController()
     public static var sharedInMemory: DataController = InMemoryDataController()
     
+    /// A possible hacky solution to prevent  #2185 crashes.
+    public func lazyInitialization() {
+        DispatchQueue.main.async {
```

Comment:
Does this require async? or can we call it in AppDelegate normally without the dispatch?

Reply:
yeah, removed it